tester/src/test/scala/spinal/tester/scalatest/LatchTester.scala

```diff
@@ -0,0 +1,39 @@
+package spinal.tester.scalatest
+
+import org.scalatest.funsuite.AnyFunSuite
+import spinal.core._
+import spinal.core.sim._
+
+class LatchTester extends AnyFunSuite {
+  test("bad latch") {
+    SimConfig.compile(new Component {
```

Comment:
I guess this test will fail, and the failure should be asserted here.

Maybe using something like:

```scala
var goodFail = false
try {
  SpinalVerilog(new Component …)
} catch {
  case e: ExpectedErrorType => goodFail = true
}
assert(goodFail)
```

Reply:
It failed as expected, I got this working:

```scala
  test("bad latch") {
    var goodFail = false

    try {
      SpinalVerilog(new Component {
        val myCond = Bool
        val inVal = Bits(8 bits)
        val latchVal = Bits(8 bits)

        when(myCond) {
          latchVal := inVal
        }
      })
    } catch {
      case e: SpinalExit if e.getMessage().contains("LATCH DETECTED") =>
        goodFail = true
    }

    assert(goodFail)
  }
```